src/resmom/catch_child.c

```diff
@@ -1159,13 +1163,17 @@ update_jobs_status(void)
 
 	for (pjob = (job *)GET_NEXT(svr_alljobs);
 		pjob; pjob = (job *)GET_NEXT(pjob->ji_alljobs)) {
+			time_now = time(NULL);
+			if (pjob->ji_last_resc_updated >= (time_now - min_check_poll))	
```

Comment:
I think you should set time_now once, just before you enter the loop.

Again, I think the logic should be:
```suggestion
			if (pjob->ji_last_resc_updated < (time_now - min_check_poll))	
```